packages/core/craco.config.js

```diff
@@ -1,6 +1,42 @@
 const webpack = require('webpack');
 
 module.exports = {
+  jest: {
```

Comment:
I am confused about the Jest config. I see we have a jest.config.js in this folder. Should we delete the jest config in Craco?